assistant_dists/dream_kg/docker-compose.override.yml

```diff
@@ -219,6 +219,7 @@ services:
     command: flask run -h 0.0.0.0 -p 8020
     environment:
       - FLASK_APP=server
+      - CUDA_VISIBLE_DEVICES=0
```

Comment:
do you have this info in your component and containers cards?

Reply:
reverted